lib/rhc/commands/snapshot.rb

```diff
@@ -30,7 +30,12 @@ def save(app)
 
       raise RHC::DeploymentsNotSupportedException.new if options.deployment && !rest_app.supports?("DEPLOY")
 
-      filename = options.filepath ? options.filepath : "#{rest_app.name}.tar.gz"
+      if options.filepath and File.exists? options.filepath
+        return 0 unless RHC::Helpers.agree "File #{options.filepath} already exists. Do you want to overwrite this file? (yes|no): "
+        filename = options.filepath
+      else
+        filename = options.filepath ? options.filepath : "#{rest_app.name}.tar.gz"
+      end
```

Comment:
If the user does not specify the file path, so the default of `#{rest_app.name}.tar.gz` is chosen, and a file by that name exists, should we not print the same warning? Perhaps the logic should be more like the following:

```ruby
      filename = options.filepath || "#{rest_app.name}.tar.gz"
      if File.exists? filename
        return 0 unless RHC::Helpers.agree "yada yada"
      end
```